### ISSUE TITLE
Docker Image

### ISSUE THREAD

**Issue by Manouchehri** [mechaphish/setup#1]
Would be cool to have a Dockerfile/image. Installing mechaphish isn't very fun at the moment. =)

``` IRC
9:35:59 PM <dave0x6d> zardus: is it okay if I open up a GitHub ticket about it so I can subscribe to it?
9:36:06 PM <dave0x6d> seems easier than me asking you every week.
9:36:35 PM <@zardus> absolutely
9:36:38 PM <@zardus> on mechaphish/setup
```

**Comment by brianherman** [comment]
Does the docker file install everything? It seems to I will create a pull request and add to the documentation.

**Comment by Manouchehri** [comment]
Closed, 66a34288ba50209bb5fae06adbab5d884c8d6cb2 added a Dockerfile.
